src/components/pages/ChildDashboard/RenderChildDashboard.js

```diff
@@ -9,10 +9,20 @@ import { HUD } from '../HeadsUpDisplay/index';
 import Explosion from '../../../assets/images/gamemodeimg/explosion.png';
 import change_your_avatar from '../../../assets/images/child_dashboard_images/change_your_avatar.svg';
 import leaderboard_icon from '../../../assets/images/child_dashboard_images/leaderboard_icon.png';
+import { getSubmissions } from '../../../api/index';
 
 const RenderChildDashboard = props => {
   const { push } = useHistory();
   const [modalVisible, setModalVisible] = useState(false);
+  const [submissionData, setSubmissionData] = useState([]);
+
+  // ChildId hardcoded for now until global state of child is retrived
```

Comment:
Maybe use a variable for childID instead of just putting '1', so that even though it's hardcoded, you could have a consistent child ID throughout the entire file, with one area you can change it. But it probably doesn't matter much if it's going to be replaced with global state eventually.

Reply:
That's a good suggestion, Isaac.  That way, when the state is hooked up, you'll only need to change the variable, and not risk missing any of the 1's if they're used anywhere else.